config/config.go

```diff
@@ -50,9 +51,9 @@ func LoadConfigFile(path string) (*Config, error) {
 func LoadConfigDev() (*Config, chan struct{}, string, error) {
 	// start a vault dev instance
 	shutdownCh := initDevVaultCore()
-
+	time.Sleep(10 * time.Second)
```

Comment:
Sleeping for some time between the goroutine and setting up vault is a bandaid. A proper fix would be to ping every half a second to see if vault is up, and if it's not up in 5 seconds, return error. 5 seconds being a random number I pulled out of thin air